SECURITY.md

```diff
@@ -1,43 +0,0 @@
-# Security Policy
```

Comment:
This file shouldn't be here. There's a problem with your branch.

Reply:
Thanks, I am fixing now!

Reply:
I deleted the file, but I had to return it. This file was created in a "git pull" that  I executed and without it the github is giving errors.

Reply:
This deletes the file though! Try a rebase.

Reply:
It is fixed in the last pull! Thanks!